GetIntoTeachingApi/Jobs/LocationSyncJob.cs

```
@@ -149,13 +149,18 @@ private async Task<string> RetrieveCsv(string ukPostcodeCsvUrl)
 
             try
             {
+                ZipFileChecker.AssureNoBombs(zipPath);
                 ZipFile.ExtractToDirectory(zipPath, csvPath);
-                _logger.LogInformation($"LocationSyncJob - CSV Extracted");
+                _logger.LogInformation("LocationSyncJob - CSV Extracted");
+            }
+            catch (BombFoundException bombFoundException)

```

I would let this bubble up so we get a notification in Sentry